gravitee-apim-console-webui/src/management/api/creation-v4/api-creation-v4.component.spec.ts

```diff
@@ -1274,6 +1273,7 @@ describe('ApiCreationV4Component', () => {
   describe('step5', () => {
     const API_ID = 'my-api';
     const PLAN_ID = 'my-plan';
+    let step6Harness: Step5SummaryHarness;
```

Comment:
Strange to have a `Step5SummaryHarness` instance named `step6Harness` 🤔

Reply:
We used to have a "documentation Step". When we removed it, the Step6 object became the Step5 object but some variables have not been updated.

Reply:
It's because the Documentation step is no longer in the project 🙃 It was named when Documentation was Step 5 and the summary was step 6